.env

```diff
@@ -0,0 +1,8 @@
+# Default env variables used by docker-compose
```

Comment:
We would get away with not having this file and just use [variable substitution](https://docs.docker.com/compose/compose-file/compose-file-v3/#variable-substitution) within the docker-compose file. So, like we could do `mysql:${MYSQL_VERSION:-8}`, etc.

Reply:
Noice! I thought the compose file wouldn't support such thing